nodejs/eks/cluster.ts

```diff
@@ -500,16 +500,6 @@ export class Cluster extends pulumi.ComponentResource {
      */
     public readonly instanceRole: pulumi.Output<aws.iam.Role>;
 
-    /**
-     * The security group for the cluster's nodes.
-     */
-    public readonly nodeSecurityGroup: aws.ec2.SecurityGroup;
-
-    /**
-     * The ingress rule that gives node group access to cluster API server
-     */
-    public readonly eksClusterIngressRule: aws.ec2.SecurityGroupRule;
```

Comment:
This is an API breaking change.  It's somewhat unlikely this is being used though - and it should in practice be easy enough to work around by referring to the properties on the nodegroup.  But may be worth calling out as breaking change in the `CHANGELOG`.